src/releasers/terraform-module.ts

```diff
@@ -85,6 +86,19 @@ export class TerraformModule extends ReleasePR {
         packageName: this.packageName,
       })
     );
+    // Update versions.tf to current candidate version.
+    // A module may have submodules, so find all versions.tf to update.
+    const versionFiles = await this.gh.findFilesByFilename('versions.tf');
```

Comment:
The one annoying caveat here, is I believe GitHub doesn't index files until they've been merged for a little while -- you might bump into occasional hiccups as new `versions.tf` files are added.

Reply:
Ack, it seems to be a couple of minutes but most of our repos have this so its hopefully indexed.